File: python/triple_double.py
def triple_double(num1, num2):
	for i in range (0,10):
		if str(i)*3 in str(num1):
			if str(i)*2 in str(num2):
				return 1
	return 0

print(triple_double(451999277, 41177722899), 1)
print(triple_double(1222345, 12345), 0)
print(triple_double(12345, 12345), 0)
print(triple_double(666789, 12345667), 1)
print(triple_double(10560002, 100), 1)

File: nodejs/triple_double.js
function tripledouble(num1, num2)
{
	num1 = String(num1);
	num2 = String(num2);
	for(var i = 0; i < 10; i++)
		if(num1.lastIndexOf(i+""+i+""+i) >= 0)
			if(num2.lastIndexOf(i+""+i) >= 0)
				return 1;
	return 0;
}

console.log(triple_double(451999277, 41177722899), 1)
console.log(triple_double(1222345, 12345), 0)
console.log(triple_double(12345, 12345), 0)
console.log(triple_double(666789, 12345667), 1)
console.log(triple_double(10560002, 100), 1)
